bin/input-remapper-gtk

```diff
@@ -24,27 +24,15 @@
 
 import sys
 import atexit
-import gettext
-import locale
-from inputremapper.configs.data import get_data_path
-import os.path
 from argparse import ArgumentParser
+from inputremapper.gui.gettext import _, LOCALE_DIR
```

Comment:
```suggestion
from inputremapper.gui.gettext import _
```

Reply:
Please note that `LOCALE_DIR` is used in the debug message.

Reply:
ah, whoops